migrate/Cargo.toml

```
@@ -0,0 +1,31 @@
+[package]
+name = "fnn-migrate"
+version = "0.2.1"
+edition = "2021"
+build = "build.rs"
+
+[dependencies]
+rocksdb = { package = "ckb-rocksdb", version = "=0.21.1", features = [
+    "lz4"
+], default-features = false }
+
+tracing = "0.1"
+tracing-subscriber = { version = "0.3", features = ["env-filter"] }
+indicatif = "0.16"
+console = "0.15.8"
+bincode = "1.3.3"
+thiserror = "1.0.58"
+serde = { version = "1.0.197", features = ["derive"] }
+hex = "0.4.3"
+clap = { version = "4.0", features = ["derive"] }
+fiber = { package = "fnn", path = "../" }
+serde_json = "1.0.135"
+
+fiber_v020 = { package = "fnn", git = "https://github.com/nervosnetwork/fiber.git", tag = "v0.2.0" }
+fiber_v021 = { package = "fnn", git = "https://github.com/nervosnetwork/fiber.git", tag = "v0.2.1" }
+fiber_v022 = { package = "fnn", git = "https://github.com/nervosnetwork/fiber.git", revision = "e38b0b3cac8cfaf9b324b6e25612ff7f09deb193" }
```

I pushed a tag [v0.3.0-rc1](https://github.com/nervosnetwork/fiber/releases/tag/v0.3.0-rc1), please change it to:
```suggestion
fiber_v030 = { package = "fnn", git = "https://github.com/nervosnetwork/fiber", tag = "v0.3.0-rc1" }
```